src/inc/msquic_winuser.h

```diff
@@ -19,6 +19,14 @@
 #ifndef _MSQUIC_WINUSER_
 #define _MSQUIC_WINUSER_
 
+#ifndef WIN32_LEAN_AND_MEAN
+#define WIN32_LEAN_AND_MEAN
+#endif
+
+#ifndef SECURITY_WIN32
+#define SECURITY_WIN32
+#endif
```

Comment:
You sure this is necessary? What requires this define?

Reply:
We define it in the build setup, and googling it looks like its got something to do with SSPI

https://docs.microsoft.com/en-us/windows/win32/secauthn/using-sspi-with-a-windows-sockets-client

I'm not exactly sure what its for, I was just going off of the build system.

Reply:
The define shouldn't be required externally by apps that consume our API. The defines in this file should be primarily for external consumers.

Reply:
Removed then. I didn't know exactly what it was for.